integration/test/README.md

```diff
@@ -44,19 +26,37 @@ application to wrap target job execution.
               $PATH_TO_DDT_CLIENT_SESSION_FILE" \
               ./geopm_test_integration.py -v TestIntegration.test_count`
 
-EXAMPLES
---------
-Since the log files that are emitted from the individual tests are
-opened and appended to on subsequent runs, it may become necessary to
-periodically clean up the files to avoid conflating experimental data.
-```git clean``` can be used in this instance to clean up all of the
-temporary files:
-
-    $ git clean -fd .
-    $ export GEOPM_RUN_LONG_TESTS=true
-    $ ./geopm_test_integration.py -v TestIntegration.test_power_consumption
-
-Or as a one-liner:
-
-    $ git clean -fd . ; GEOPM_RUN_LONG_TESTS=true ./geopm_test_integration.py -v TestIntegration.test_power_consumption
-
+EXAMPLE
+-------
+
+```
+#!/bin/bash
+
+# Set to geopm source code location
```

Comment:
I expected to see this here:
```
if [ -f ${HOME}/.geopmrc ]; then
    source ${HOME}/.geopmrc
    source ${GEOPM_SOURCE}/config/run_env.sh
else
    exit -1
fi
```
Then you don't need to hardcode the other env vars below.

Reply:
I'm just trying to keep the instructions self contained and simple.  I'm not sure referring to two other bash scripts, one of which is not part of our repository, is helpful.